projects/go-lib/src/lib/components/go-side-nav/go-nav-item/go-nav-item.component.ts

```diff
@@ -1,14 +1,29 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { NavItem } from '../nav-item.model';
 import { GoSideNavService } from '../go-side-nav/go-side-nav.service';
+import { GoConfigService } from '../../../go-config.service';
+import { GoConfigInterface } from '../../../go-config.model';
+import { distinctUntilKeyChanged } from 'rxjs/operators';
```

Comment:
@StevenUlmer we can probably remove this line if we're not going to implement `distrinctUntilKeyChanged`

Reply:
Actually, did you `pipe` it? 
```
this.configService.config.pipe(
  distinctUntilKeyChanged('brandColor')
).subscribe((value: GoConfigInterface) => {
  this.theme = value.brandColor;
});
```

Reply:
I did. It ended up having something to do with how I was updating the config in the service. It works now